backoffice/src/utils/dateTime.ts

```diff
@@ -25,7 +25,7 @@ export const customDateStringFormat = (
   const date = moment(dateTimeString);
 
   if (!date.isValid()) {
-    return "";
+    return typeof dateTimeString === "string" ? dateTimeString : "";
```

Comment:
so if there's something wrong with the date, we're checking if the type of the string representation of the date is a string, and if it is, returning that; but if it's something else, returning empty string. Have I got that right?

Reply:
So if the date parsed from it is invalid then we'll still return an invalid date?

Reply:
Yeah absolutely this, we can't guarantee it'll be in a correct date format. Currently we're just returning nothing if it isn't, which is effectively stripping data out